recipes/orekit/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "orekit" %}
+{% set version = "9.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://www.orekit.org/git/orekit-labs/python-wrapper-main.git
+  git_tag: orekit-v9
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win32 or linux32]
+
+  number: 0
+
+  # script: python setup.py install --single-version-externally-managed --record record.txt
+
+  rpaths:  # Only valid for linux
```

Comment:
The comment can be dropped unless it is helpful to you. If so, would move to the line directly above.

Reply:
Removed.